client/js/views/Rules.vue

```diff
@@ -286,7 +297,8 @@ export default {
           {
             title: 'Scuttle',
             icon: 'skull-crossbones',
-            description: 'Scrap an opponent’s point card with a bigger one from your hand',
+            description:
+              'Scrap an opponent’s point card with a bigger one from your hand.  Ties are broken by the suit: ♣️ (weakest) < ♦️ < ♥️ < ♠️ (strongest).',
```

Comment:
```suggestion
              'Scrap an opponent’s point card with a bigger one from your hand.  Ties are broken by the suit: &clubs; (weakest) &lt; &diams; &lt; &hearts; &lt; &spades; (strongest).',
```
I actually think these should be the html entities instead. I don't care enough to have it be a blocker since it's arguably easier to read this way, but since this _is_ html, the encoding makes more sense to me. Same with the less than symbols.

It's arguably more valid and looks the same
| html |
|-------|
| ![image](https://user-images.githubusercontent.com/414475/195697198-1a7697e6-f194-4e51-9e8f-6246752c16fb.png) |

Reply:
At the very least we should encode the less than symbols, emojis I'm fine with either considering the ease of readibility with the images. 
![image](https://user-images.githubusercontent.com/414475/195699555-98e5f9d0-c594-401d-9e02-1778a78abb6f.png)

Reply:
@seriouslysean it looks like this is native vue behavior:
https://codepen.io/aleph_one/pen/gOzyYrb

Reply:
Makes sense since we’re not using `v-html` I suppose. Thanks for the info!